vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "What are MMRMs? A brief introduction"
+package: mmrm
+#bibliography: '`r system.file("REFERENCES.bib", package = "mmrm")`'
```

Comment:
please use same header/layout as e.g. in https://github.com/openpharma/mmrm/blob/main/vignettes/satterthwaite.Rmd. In particular please add references to the bib file

Reply:
Done